action/main.ts

```diff
@@ -148,19 +145,24 @@ async function run() {
       ignoreLastBuildOnBranch: maybe(ignoreLastBuildOnBranch),
     });
 
-    const [{ url, code }] = await Promise.all([chromatic]);
+    const output = await chromatic;
 
-    setOutput('url', url);
-    setOutput('code', code.toString());
+    setOutput('url', output.url);
+    setOutput('buildUrl', output.buildUrl);
+    setOutput('storybookUrl', output.storybookUrl);
+    setOutput('code', output.code.toString());
 
-    if (code !== 0) {
+    if (output.code !== 0) {
       setFailed('non-zero exit code');
     }
+
+    process.exit(output.code);
```

Comment:
here's the fix

Reply:
inspired from:
https://github.com/chromaui/chromatic-cli/blob/906f4426fc2641e78b556b1c1117d9b1e5301866/bin/main.js#L33-L36